Source/relay/TourGuide/Items of the Month/2023/Rock Garden.ash

```diff
@@ -0,0 +1,92 @@
+string gravelMessage(int gravels)
```

Comment:
i think this is fine as a starting point but at some point i'd like to abstract this out a bit more; i don't think we -really- need a whetstone reminder, but gravel should be within the freekills tile and milestone in desert means it doesn't really need to be here in great quantity either. but overall the logic looks right and the tile is fine, and worth pushing up so there's support even if i'd like to soup the support up.